### PR TITLE
Added metrics

### PR DESCRIPTION
**Pull request by qwe123coder**
I added the yml code for GitHub metrics
@eddiejaoude you would have to create a Personal access tokens from here https://github.com/settings/tokens/new

and then give it admin:org
image

and make secrets and name it  METRICS_TOKEN

### CODE REVIEW

**Comment by schmelto** [comment]
why removed the profile readme?

**Comment by eddiejaoude** [comment]
@qwe123coder could you add an example screenshot please

**Review by Vyvy-vi — COMMENTED** [review]
Thanks for opening this PR!
I had a question- Why was `profile/README.md` deleted, and what's the format for this action? (Can you share a screenshot of what you expect this to output, by running the action on your fork?)

**Comment by qwe123coder** [comment]
> @qwe123coder could you add an example screenshot please

screencapture-metrics-lecoq-io-2021-09-20-15_53_37 (1)

**Comment by qwe123coder** [comment]
> Thanks for opening this PR!
> I had a question- Why was `profile/README.md` deleted, and what's the format for this action? (Can you share a screenshot of what you expect this to output, by running the action on your fork?)



> > @qwe123coder could you add an example screenshot please
> 
> screencapture-metrics-lecoq-io-2021-09-20-15_53_37 (1)

**Comment by schmelto** [comment]
I#m not sure if we need this information on the profile readem for the organization. Maybe someone else can share some thoughts.

**Comment by qwe123coder** [comment]
it will look cool

**Comment by CodingSpecies** [comment]
@qwe123coder @schmelto I agree it will look good, it uses this link:

https://metrics.lecoq.io/

but if we were going to use this, we might have to disable some of the features, so the README isn't so long. If the readme was long, it wouldn't seem so appealing to viewers, and they might lose interest to seeing what comes after scrolling down the readme(after the readme comes the pinned repos).

That is what I think, but like @schmelto said, we should see what others also think.

File: workflows/readme.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - uses: lowlighter/metrics@latest
+        with:
+          # Your GitHub token
+          token: ${{ secrets.METRICS_TOKEN }}
+
+          # Options
+          user: EddieHubCommunity
+          template: classic
+          base: header, activity, community, repositories, metadata
+          config_timezone: Asia/Calcutta
```
Comment (Akshu-on-github) [inline workflows/readme.yml:22]: And here too

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please check the above comments

**Pull request closed without merging**